buildchain/buildchain/salt_tree.py

```diff
@@ -335,7 +335,8 @@ def _run(self) -> None:
     Path('salt/metalk8s/repo/init.sls'),
     Path('salt/metalk8s/repo/installed.sls'),
     Path('salt/metalk8s/repo/macro.sls'),
-    Path('salt/metalk8s/repo/offline.sls'),
+    Path('salt/metalk8s/repo/CentOS.sls'),
```

Comment:
Filenames are all-lowercase.

Reply:
Change done. (now named rhel.sls for the old CentOS.sls and debien.sls for old Ubuntu.sls)